docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -19,6 +19,8 @@ To integrate Codacy with your Git workflow, follow these steps:
 
 ## 1. Configuring the quality gate rules {: id="configuring-gate"}
 
+<!--TODO PLUTO-505 Link content with gate policies-->
```

Comment:
@claudiacarpinteiro, consider updating the Getting Started tutorial in the scope of a separate (and subsequent) documentation task, if it helps to reduce the scope of the initial updates for the new feature.

Reply:
The updates to the Getting Started ended up being just a small change in one sentence. 😄